src/commcare_cloud/ansible/roles/pgbouncer/templates/pgbouncer2.ini.j2

```diff
@@ -0,0 +1,221 @@
+;;; {{ansible_managed}}
```

Comment:
how is this file different from `pgbouncer.ini.j2`? If it's not can they just use the same template?

Reply:
`logfile`, `pidfile`, `unix_socket_dir` as per `so_reuseport` instructions in https://www.pgbouncer.org/config.html